src/dmd/iasmdmd.d

```diff
@@ -4461,24 +4461,23 @@ void asm_primary_exp(out OPND o1)
  * Params:
  *      e =     Input. The expression to evaluate. This can be an arbitrarily complex expression
  *              but it must either represent a constant after CTFE or give a higher level variable.
- *      o1 =    Output. The ASM operand to define from `e`.
- *      s =     Output. The symbol when `e` represents a variable.
+ *      o1 =    if `e` turns out to be a constant, `o1` is set to reflect that
+ *      s =     if `e` turns out to be a variable, `s` is set to reflect that
  *
  * Returns:
  *      `TOK.variable` if `s` was set to a variable,
  *      `TOK.const_` if `e` was evaluated to a valid constant,
  *      `TOK.error` otherwise.
  */
-TOK tryExpressionToOperand(Expression e, ref OPND o1, ref Dsymbol s)
+TOK tryExpressionToOperand(Expression e, out OPND o1, out Dsymbol s)
```

Comment:
`o1` cannot be `out` otherwise it will be reset

Reply:
I'm not so sure about that. I found another cause for the test failures. This sort of thing, though, is exactly what I want to clear up.